Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -190,12 +194,16 @@ public struct PackageBuilder {
         return true
     }
 
+    private var rerootedManifestPath: AbsolutePath {
+        return AbsolutePath.root.appending(component: Manifest.filename)
```

Comment:
I think we should factor out the "root path" within the file system view we are given (which would just be `AbsolutePath.root`, currently).

The reason is that I hope eventually to allow multiple packages inside one repository, and I think doing this now would clean up this patch versus the hard-coded `AbsolutePath.root` references.